server/pulp/server/webservices/views/consumer_groups.py

```diff
@@ -0,0 +1,373 @@
+import json
+
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect, HttpResponseBadRequest
+from django.views.generic import View
+
+from pulp.server import exceptions as pulp_exceptions
+from pulp.server.auth import authorization
+from pulp.server.db.model.consumer import ConsumerGroup
+from pulp.server.db.model.criteria import Criteria
+from pulp.server.managers import factory
+from pulp.server.managers.consumer.group.cud import bind, unbind
+from pulp.server.webservices.controllers.decorators import auth_required
+from pulp.server.webservices.controllers.base import json_encoder as pulp_json_encoder
```

Comment:
This and line 13 should be swapped for alphabetical imports since base comes before decorators.